internal/testutil/helpers.go

```diff
@@ -30,3 +33,13 @@ func RandomName(prefix ...string) string {
 	}
 	return string(b)
 }
+
+func SkipUntil(t TestingT, date string) {
+	deadline, err := time.Parse(time.DateOnly, date)
+	require.NoError(t, err)
+
+	if time.Now().Before(deadline) {
+		fmt.Printf("Skipping test until %s. Time right now: %s\n", deadline.Format(time.DateOnly), time.Now())
```

Comment:
This must use `t.Logf`. Direct to stdout means leaking in the output.

Reply:
or t.Skipf to combine with the line below.

Reply:
For my knowledge, where does this leak affect us? Would this pollute the nightly runner output?

Reply:
AFAIK it would go to stdout and be unattributable to the test.